includes/core/classes/class-setup.php

```diff
@@ -216,6 +217,41 @@ private function maybe_create_flush_rewrite_rules_flag(): void {
 		}
 	}
 
+	/**
+	 * Adds a privacy policy statement.
+	 *
+	 * Every plugin that collects, uses, or stores user data,
+	 * or passes it to an external source or third party,
+	 * should add a section of suggested text to the privacy policy postbox.
+	 *
+	 * This is best done with wp_add_privacy_policy_content( $plugin_name, $policy_text ).
+	 * This will allow site administrators to pull that information into their site’s privacy policy.
+	 *
+	 * The HTML contents of the $content supports use of a specialized .privacy-policy-tutorial CSS class
+	 * which can be used to provide supplemental information.
+	 * Any content contained within HTML elements that have the .privacy-policy-tutorial CSS class applied
+	 * will be omitted from the clipboard when the section content is copied.
+	 *
+	 * @see https://developer.wordpress.org/plugins/privacy/suggesting-text-for-the-site-privacy-policy/
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return void
+	 */
+	public function add_privacy_policy_content() {
+		if ( ! function_exists( 'wp_add_privacy_policy_content' ) ) {
+			return;
+		}
+		$content = '<h2>' . __( 'Inform your visitors about GatherPress\' use of OpenStreetMap services.', 'gatherpress' ) . '</h2>'
+				. '<p><strong class="privacy-policy-tutorial">' . __( 'Suggested Text:', 'default' ) . '</strong> '
```

Comment:
Fix text domain

Reply:
No, it's intentional. 

GatherPress does not need to stress the translation contributors with defaults that are translated a thousand times.

Reply:
Are you saying PHPCS complains if you leave it off and WP Plugin Guidelines complains if it's not "gatherpress"? Maybe don't include it and if PHPCS complains, add a PHPCS Ignore.

Reply:
> Maybe don't include it and if PHPCS complains, add a PHPCS Ignore.

Ok, I did exactly this in my last 2 commits. Lets see what the guidelines check, will say ...

Reply:
<img width="1983" height="1269" alt="image" src="https://github.com/user-attachments/assets/cab6ad16-43ea-4016-8be9-64a094bb5c05" />

Can you restart the workflow please @mauteri ?

Reply:
Thanks.

But this keeps being suspicious, the wp.org plugin directory guidelines are now constantly failing with the error shown in the screenshot, which seems to be totally unrelated.

Any ideas @mauteri ?

Reply:
@carstingaxion Try changing tested up to 6.8.1 to just 6.8. Seems that is what it is erroring on (which is new, as it's been that way for awhile).

Reply:
If that fixes it, we can put in a PR to change that and pull it into all PRs we're working on.

Reply:
> So next, I'm going top re-add the 'default' textdomain and remove the phpcs:ignore statement.

Which worked out very well, with no errors.

Do you still need additional information on the _Why & How_ of this idea or are you able to merge it?